polars/series/index.ts

```diff
@@ -1528,16 +1531,20 @@ export function _Series(_s: any): Series {
       return this.length;
     },
     lt(field) {
-      return dtypeWrap("Lt", field);
+      if (typeof field === "number") return dtypeWrap("Lt", field);
+      return wraps("lt", field);
```

Comment:
the extra method seems unnecessary, also shouldn't we check that this is a Series

```ts
    if (typeof field === "number") {
        return dtypeWrap("Lt", field);
    }
    else if (Series.isSeries(field)) {
        return wrap("Lt", field._s)
    } else {
        throw new Error()
    }
```

Reply:
I was not sure how to handle `...args` and `args` in one func without throwing a TS error.
We could check if it's a Series, but I think that's implied by default.

Reply:
Would this be better? `return wrap("lt", (field as any)._s);`